test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/NpgsqlCommandTests.cs

```diff
@@ -0,0 +1,40 @@
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    public class NpgsqlCommandTests : TestHelper
+    {
+        public NpgsqlCommandTests(ITestOutputHelper output)
+            : base("Npgsql", output)
+        {
+        }
+
+        [Theory]
+        [MemberData(nameof(PackageVersions.SqlClient), MemberType = typeof(PackageVersions))]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
```

Comment:
I recommend removing the "RunOnWindows" since we haven't figured out the docker situation on our Windows CI builds.

Reply:
```suggestion
```